python/requirements/development.txt

```diff
@@ -10,7 +10,7 @@ h5py>=2.6.0
 jsonschema>=3.0.0
 jupyter-book>=0.12.1
 kastore
-lshmm
+#lshmm
```

Comment:
We should probably still keep this in the development requirements?

Reply:
Yep, reinstated.